api/Attr.json

```diff
@@ -52,34 +52,28 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Attr/localName",
           "support": {
             "chrome": {
-              "version_added": "46",
-              "notes": "This API was previously available on the <a href='https://developer.mozilla.org/docs/Web/API/Node'><code>Node</code></a> API."
+              "version_added": "1"
             },
             "chrome_android": {
-              "version_added": "46",
-              "notes": "This API was previously available on the <a href='https://developer.mozilla.org/docs/Web/API/Node'><code>Node</code></a> API."
+              "version_added": "18"
             },
             "edge": {
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "48",
-              "notes": "This API was previously available on the <a href='https://developer.mozilla.org/docs/Web/API/Node'><code>Node</code></a> API."
+              "version_added": "1"
             },
             "firefox_android": {
-              "version_added": "48",
-              "notes": "This API was previously available on the <a href='https://developer.mozilla.org/docs/Web/API/Node'><code>Node</code></a> API."
+              "version_added": "4"
             },
             "ie": {
               "version_added": false
             },
             "opera": {
-              "version_added": "33",
-              "notes": "This API was previously available on the <a href='https://developer.mozilla.org/docs/Web/API/Node'><code>Node</code></a> API."
+              "version_added": "≤12.1"
             },
             "opera_android": {
-              "version_added": "33",
-              "notes": "This API was previously available on the <a href='https://developer.mozilla.org/docs/Web/API/Node'><code>Node</code></a> API."
+              "version_added": "≤12.1"
             },
             "safari": {
               "version_added": "1.3"
```

Comment:
I did not try to confirm that the `Attr` members were supported since 1.3 as opposed to 1. It seems pretty likely that they were supported at the same time on `Attr` and `Element` since the support was via `Node`, but it doesn't seem worthwhile trying to find the right answer.

Reply:
Unfortunately, the failing lint means the distinction can't be ignored. Looking into it...